docs/src/App/Home/Home.tsx

```diff
@@ -1,19 +1,22 @@
-import React, { Fragment } from 'react';
+import React, { Fragment, ReactNode } from 'react';
 import { Link } from 'react-router-dom';
 import { Box, Button, Text } from '../../../../lib/components';
-import { BoxProps } from '../../../../lib/hooks/useBox';
 import { ConfigConsumer } from '../ConfigContext';
 import { Logo } from '../Logo/Logo';
 import { Github } from './Github/Github';
 import * as styles from './Home.treat';
 
-const actionProps: BoxProps = {
-  display: 'flex',
-  width: 'full',
-  marginLeft: ['none', 'xxsmall'],
-  marginRight: ['none', 'xxsmall'],
-  paddingBottom: ['xsmall', 'none'],
-};
+const Action = ({ children }: { children: ReactNode }) => (
+  <Box
+    display={'flex'}
+    width={'full'}
```

Comment:
`width="full"` ?

Reply:
Sorry, i was relying on prettier fixing my multi-cursor refactor laziness haha

Reply:
I assumed that'd be the case